layouts/_default/baseof.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<title>Git for Windows</title>
+		<meta name="description" content="We bring the awesome Git VCS to Windows">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300,400,700">
+		<link rel="stylesheet" href="css/pack.css">
+		<style type="text/css">
+			body {
+				color: #FFF;
+			}
+
+			h1 {
+				text-align: center;
+			}
+
+			article p,h3,ul {
+				margin-left: auto;
+				margin-right: auto;
+				width: 65%;
+			}
+		</style>
+		<link rel="shortcut icon" href="favicon.ico">
+		<!--[if lt IE 9]>
+		<script src="js/html5shiv.js"></script>
+		<script src="js/respond.js"></script>
+		<![endif]-->
```

Comment:
Do we expect people who want Git for Windows to be running IE < 9 (released in tandem with Windows 7)?

Reply:
No, and Hugo even skips this completely. I guess we can drop this.